Gemfile

```diff
@@ -39,7 +39,7 @@ gem 'puma', '~> 3.12'
 # Use Redis adapter to run Action Cable in production
 # gem 'redis', '~> 3.0'
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '~> 5.1.5'
+gem 'rails', '~> 5.2.4.2'
```

Comment:
We should avoid freezing very minor versions. They should not change interfaces and allows to always install the latest security fixes. Ideally we should be making it a bit less specific `major.minor`...
Any reason why we need so much specificity?

Reply:
It's not needed, I just inserted the latest version. Good point